Google.Api.Generator.Tests/ProtoTests/Showcase/Google.Showcase.V1Beta1/PackageApiMetadata.g.cs

```diff
@@ -25,7 +25,7 @@ namespace Google.Showcase.V1Beta1
     internal static class PackageApiMetadata
     {
         /// <summary>The <see cref="gaxgrpc::ApiMetadata"/> for services in this package.</summary>
-        internal static gaxgrpc::ApiMetadata ApiMetadata { get; } = new gaxgrpc::ApiMetadata("Google.Showcase.V1Beta1", GetFileDescriptors);
+        internal static gaxgrpc::ApiMetadata ApiMetadata { get; } = new gaxgrpc::ApiMetadata("Google.Showcase.V1Beta1", GetFileDescriptors).WithRequestNumericEnumJsonEncoding(true);
```

Comment:
Is Showcase run for REST?

Which makes me think, should we fail if ApiMetada has this setting in true, but only supports gRPC?

Reply:
We don't actually run it at all - but yes, we generate it for REST.
There's a bit of a disconnect though - this is at the ApiMetadata level, but transport support is at the ServiceMetadata level. That may have been a mistake, of course... but we can't easily fix it now, and it makes checking things tricky.

Reply:
Yes, that makes sense.